packages/web/src/scss/components/Timeline/_theme.scss

```diff
@@ -20,3 +24,25 @@ $timeline-heading-padding: tokens.$space-600;
 
 $timeline-content-padding-bottom: tokens.$space-900;
 $timeline-content-padding-top: tokens.$space-600;
+
+$size-dictionary: dictionaries.$size;
+$size-config: (
+    small: (
+        marker-size: $timeline-marker-size,
+        icon-size: $timeline-icon-size,
+        dot-size: $timeline-marker-dot-size,
```

Comment:
Doesn't it make sense to hard-code these values now when we have sizes? Or are the variables used elsewhere?